app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -190,6 +201,53 @@ class StudentForm extends React.Component {
     );
   }
 
+  createStripeCustomer() {
```

Comment:
what happens if the customer is already created but the rest of the student form submission fails? if the student clicks submit, won't a new customer be created again